eclass/bitcoincore.eclass

```diff
@@ -179,7 +181,7 @@ if [ "${BITCOINCORE_NEED_LIBSECP256K1}" = "1" ]; then
 	BITCOINCORE_COMMON_DEPEND="${BITCOINCORE_COMMON_DEPEND} $LIBSECP256K1_DEPEND"
 fi
 if [ "${PN}" = "libbitcoinconsensus" ]; then
-	DEPEND="$DEPEND ${BITCOINCORE_COMMON_DEPEND}
+	BITCOINCORE_COMMON_DEPEND="${BITCOINCORE_COMMON_DEPEND}
```

Comment:
You shouldn't install tests based on `USE=test`. That flag has special meaning to Portage, and this makes it impossible for most of the users to control it properly.